JSON-RPC python tests frequently fail with OSError: [Errno 9] Bad file descriptor

Failure looks like this:
```
 ================================== FAILURES ===================================
_______________________________ test_acfactory ________________________________
[gw1] win32 -- Python 3.13.0 D:\a\deltachat-core-rust\deltachat-core-rust\deltachat-rpc-client\.tox\py\Scripts\python.EXE

acfactory = <deltachat_rpc_client.pytestplugin.ACFactory object at 0x000002049AC1DB50>

    def test_acfactory(acfactory) -> None:
        account = acfactory.new_configured_account()
        while True:
            event = account.wait_for_event()
            if event.kind == EventType.CONFIGURE_PROGRESS:
                assert event.progress != 0  # Progress 0 indicates error.
                if event.progress == 1000:  # Success
                    break
            else:
>               print(event)
E               OSError: [Errno 9] Bad file descriptor

tests\test_something.py:60: OSError
```

This looks like a problem with `print` usage in pytest when print attempts to print directly and something else (pytest?) closed the file descriptor.

pytest is configured in `tox.ini` with `log_cli = true`, maybe this changes something.

Here is CI log (from windows runner):
[job-logs.txt](https://github.com/user-attachments/files/17459106/job-logs.txt)

This is not the result of teardown, because by the time `test_acfactory` test failed with this `print` exception only 50% of tests passed and other tests continue to run successfully.

--- a/deltachat-rpc-client/tests/test_iroh_webxdc.py
+++ b/deltachat-rpc-client/tests/test_iroh_webxdc.py
@@ -7,8 +7,8 @@ If you want to debug iroh at rust-trace/log level set
     RUST_LOG=iroh_net=trace,iroh_gossip=trace
 """
 
+import logging
 import os
-import sys
 import threading
 import time
 
@@ -25,9 +25,7 @@ def path_to_webxdc(request):
 
 
 def log(msg):
-    print()
-    print("*" * 80 + "\n" + msg + "\n", file=sys.stderr)
-    print()
+    logging.info(msg)
 
 
 def setup_realtime_webxdc(ac1, ac2, path_to_webxdc):

--- a/deltachat-rpc-client/tests/test_something.py
+++ b/deltachat-rpc-client/tests/test_something.py
@@ -57,8 +57,8 @@ def test_acfactory(acfactory) -> None:
             if event.progress == 1000:  # Success
                 break
         else:
-            print(event)
-    print("Successful configuration")
+            logging.info(event)
+    logging.info("Successful configuration")
 
 
 def test_configure_starttls(acfactory) -> None:

